pages/account/manage/events.js

```diff
@@ -26,6 +26,23 @@ export async function getServerSideProps(context) {
 
   const username = session.username;
 
+  let profile = {};
+  try {
+    profile = (await getUserApi(context.req, context.res, username)).profile;
+  } catch (e) {
+    logger.error(e, `profile loading failed for username: ${username}`);
+  }
+
+  //reroute to profile deactivated page if profile is disabled
+  if(profile && profile.hasOwnProperty("isEnabled") && !profile["isEnabled"]) {
```

Comment:
`!profile["isEnabled"]` is not the coding style we generally use

I think this could be simplified to
```suggestion
  if(profile && !profile.isEnabled) {
```